adflow/mphys/mphys_adflow.py

```diff
@@ -1165,6 +1165,29 @@ def __init__(
         else:
             self.mesh_options = mesh_options
 
+        if mesh_type == "USMesh":
+            self.multi_us_mesh = False
+
+            if "multi_us_mesh_components" in self.mesh_options.keys():
+                raise TypeError(
+                    "'multi_us_mesh_components' is only for 'MultiUSMesh' mesh_type . Please don't provide any multi_us_mesh_components dictionary for 'USMesh' mesh_type."
+                )
+
+        elif mesh_type == "MultiUSMesh":
+            if "multi_us_mesh_components" not in self.mesh_options.keys():
+                raise TypeError(
+                    "The 'multi_us_mesh_components' keyword argument is *NOT* "
+                    "optional. A multi_us_mesh_components dictionary must be passed for 'MultiUSMesh' mesh_type. "
+                )
+
+            self.multi_us_mesh = True
+            # self.multi_us_mesh_instances = multi_us_mesh_instancesdict
+
+        else:
+            raise TypeError(
```

Comment:
I'm being very pedantic, but this should probably be a `ValueError` instead of a `TypeError` because the problem is that the user passed in an incorrect value

Reply:
Changed it into `ValueError`